src/components/FileContributors.tsx

```diff
@@ -101,15 +103,27 @@ const ContributorList = ({ children }: { children: React.ReactNode }) => {
 }
 
 const Contributor = ({ contributor }: { contributor: Author }) => {
+  const [isImageLoaded, setImageLoaded] = useState(false)
+
+  useEffect(() => {
+    const image = new Image()
+    image.src = contributor.avatarUrl || ""
+    image.onload = () => setImageLoaded(true)
+  }, [contributor.avatarUrl])
+
   return (
     <ListItem p={2} display="flex" alignItems="center">
-      <Avatar
-        height="40px"
-        width="40px"
-        src={contributor.avatarUrl}
-        name={contributor.name}
-        mr={2}
-      />
+      {isImageLoaded ? (
+        <Avatar
+          height="40px"
+          width="40px"
+          src={contributor.avatarUrl}
+          name={contributor.name}
+          mr={2}
+        />
+      ) : (
+        <ChakraSkeletonCircle size="10" mr={2} />
+      )}
```

Comment:
Not working as expected... 

[Chakra's Avatar](https://chakra-ui.com/docs/components/avatar) has a fallback when the image is loading, but we want to show the skeleton until the user's avatar has been fetched (Maybe @TylerAPfledderer knows?)

Reply:
@minimalsm I'm unfortunately unable to take a look at the PR locally. Something with my local does not like trying to show any of the `doc` pages that would render these avatars.

To be clear on intent: the fallback prop is only meant if the fetch returns an error, like the endpoint yields no result or if the url has a typo, etc. The skeleton should disappear when the loading is complete from the fetch regardless of the result. I think this would be good UX, else if the the skeleton stays visible when there is an error in the fetch the user might think there is something wrong with the server or browser, not the specific endpoint.

The best solution here would be to let the `Avatar` component control the loading state. Since the skeleton should disappear regardless of the outcome from the fetch, we can apply `() => setImageLoaded(true)` to both the `onLoaded` and `onError` props of the Avatar.

And to limit some of the styling duplications, the `Skeleton` can wrap the `Avatar` and pass the controlled loading state to `isLoaded`.

Check out this [Sandbox Demo](https://play.chakra-ui.com/playground?code=N4Igxg9gJgpiBcICWBbADhATgFwATFwFcBnGAZWwENsZcBfXAM0whVwHJMZKxt2AdAHaoMOfLjIBrGABsY2CIIA0uAIIA3apUz0mLNuwACYABaVJmSgFpCSAPRcefIUMiDieAMKLsmJACNCBR0AXlwACgI3XwCgrHoASlwQgD58IVxcNw9cAG0kYgBJFEoAcxgAGQhKWCgVUmxissrq2oBdZKJSCmoYcMZKGVIEl0FMrmxCTDHwjMzcAB4UufnFqVl5RSYkbG9BGn3cfyxYTAAlGqQSEP4QRkIZGVvcAqqamCgQ4AKm8rfaujLMareYLDRaTArEGZQSUFAwL7RPyBYIAOlh8LoUOhxwAHmQkAAvBG3ACMAAZbtiQcRMGBET5kXFMKjKJoqJgAKqYGRY4HQzIyVpIQSlG4gGSUQkATyp-IFdjsuAAEmzaEL3lAjrRfIRaFxStooHJiMRcBBGLgYJgWDoRUx5KYRaVcNgTLRUM1qatFP8vuEkqlcA1fi1NeFdTAEnyBfNFABRG1Yf2BtIhkp-VofCOYPXR724OxA6ELOzrOQKQTF0FFuYjQR8oQwXGiPDZPCqNBoToB5JpYBzCZTGZQpbUhZ7GIo+JI2LBL4ENkQ7kyeAcEzYbBoYjwRX+HaomTSuxQSiCKyUfyWFAQdj0Bi1+WZCeMudYAuz6eYBcFzJLjkrmu7AbluO57geR4nmeF5XnCt5KL+uAYjAQEACJnmosE3uo7AIU+8x0DGIKPiWk5MsEH6vl+P74as-7aIB66btuu52Pu2CHsep7npe14QDheGxjCcIoRw6FjKoWH8bhBaEdSJE1tW9Z8iAdBAA)

Note that I'm using `Skeleton` and not `SkeletonCircle`. The `fitContent` prop (which renders the styling `width: fit-content`) will be overridden in the `SkeletonCircle` in favor of a `boxSize` prop styling, which I think is a mistake.

Reply:
> Something with my local does not like trying to show any of the doc pages that would render these avatars.

For those encountering this, make sure `.env` doesn't have `**/docs` for `IGNORE_CONTENT=`... it's there by default in `.env.example` so this is likely common.

Reply:
Let's just allow this behaviour; it will be the default when we migrate this component in the design system. I'll push a commit to remove this logic.